time stamp
uint16[]  id_list

float64[] pwm__percent
float64[] current__mA
float64[] velocity__deg_s
float64[] position__deg
float64[] profile_vel__deg_s
float64[] profile_acc__deg_ss